libraries/AC_AutoTune/AC_AutoTune_Heli.cpp

```diff
@@ -577,9 +577,9 @@ void AC_AutoTune_Heli::load_test_gains()
             rate_test_max = orig_yaw_rate;
             accel_test_max = tune_yaw_accel;
         } else {
-            // have attitude controller use accel and rate limit parameter
-            rate_test_max = rate_max;
-            accel_test_max = accel_max;
+            // have attitude controller not perform rate limiting and angle P scaling based on accel max
+            rate_test_max = 0.0;
```

Comment:
If we are limiting the chirp then we shouldn't be hitting these limits unless there is a plot input. 

I am not very familiar with how you handle pilot input here but it may be dangerous if you don't have acceleration limiting and do accept pilot input.

Reply:
@lthall Pilot input is not allowed during the autotune input.  It works just like multi autotune where once the pilot makes inputs, the autotune maneuver is stopped.